grc/core/generator/top_block.py

```diff
@@ -64,7 +64,7 @@ def _warnings(self):
         for key in deprecated_block_keys:
             Messages.send_warning("The block {!r} is deprecated.".format(key))
 
-    def write(self):
+    def write(self, _=None):
```

Comment:
In case it's not clear: The implementer of a workflow can make use of the source of generation here, but doesn't have to (and the regular, core workflows are not changed).